docs/docs/models/depth.md

```diff
@@ -0,0 +1,94 @@
+# Depth estimation
+
+## Background
```

Comment:
Can we start with a sentence like "Our depth estimation model is used to estimate the distance an animal is from the camera, which is an input into model used to estimate animal abundance."

Reply:
good thought, thanks